hoomd/mpcd/ManualVirtualParticleFiller.h

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2009-2019 The Regents of the University of Michigan
+// This file is part of the HOOMD-blue project, released under the BSD 3-Clause License.
+
+// Maintainer: mphoward
+
+/*!
+ * \file mpcd/VirtualParticleFiller.h
+ * \brief Definition of class for backfilling solid boundaries with virtual particles.
+ */
+
+#ifndef MPCD_VIRTUAL_PARTICLE_FILLER_H_
```

Comment:
Each include guard should be unique, or else this file never gets included (may be related to comment below).
```suggestion
#ifndef MPCD_MANUAL_VIRTUAL_PARTICLE_FILLER_H_
```